src/store/store.ts

```diff
@@ -7,7 +7,15 @@ export const initialiseStore = () => {
   // the combineReducers function takes an object,
   // the key is the name we register the reducer under
 
-  return createStore(combineReducers({ list: listReducer }));
+  let preLoadedState
```

Comment:
Have you checked that this doesn't cause a crash/error when your `localStorage` is empty?